mmv1/third_party/terraform/tests/resource_logging_metric_test.go

```diff
@@ -128,11 +152,27 @@ resource "google_logging_metric" "logging_metric" {
 `, suffix, filter)
 }
 
+func testAccLoggingMetric_loggingBucket(suffix string, filter string, project_id string) string {
+	return fmt.Sprintf(`
+resource "google_logging_metric" "logging_metric" {
+  name        = "my-custom-metric-%s"
+  bucket_name = "projects/%s/locations/global/buckets/_Default"
```

Comment:
This is the format that every API / interface suggests to use. Should we have a function to auto compute this if only the bucket name is given? I guess it would be difficult to determine the bucket location...